web/controllers/skill_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule CodeCorps.SkillController do
+  use CodeCorps.Web, :controller
+
+  alias CodeCorps.Skill
+  alias JaSerializer.Params
+
+  plug :scrub_params, "data" when action in [:create]
+
+  def index(conn, params) do
+    skills =
+      case params do
+        %{"filter" => filter} ->
+          ids = String.split(filter, ",") |> Enum.map(fn(n) -> String.to_integer(n) end)
+          query = Skill |> where([p], p.id in ^ids)
+          Repo.all(query)
+        %{"query" => query} ->
+          text_query = Skill |> where([p], ilike(p.title, ^"#{query}%"))
+          Repo.all(text_query)
```

Comment:
Maybe we should keep it until we get `elasticsearch` figured out. I also am not so great with it. I ultimately used `searchkick` in Rails because I couldn't get precisely what I wanted mapped well.

Maybe we can find someone with deeper ElasticSearch skills.